include/language/en_us.lang.php

```diff
@@ -92,10 +92,10 @@
             'Releases' => 'Releases',
             'Prospects' => 'Targets',
             'Queues' => 'Queues',
-            'EmailMarketing' => 'Email Marketing',
+            'EmailTemplates' => 'Email - Templates',
             'EmailTemplates' => 'Email Templates',
```

Comment:
EmailTemplates is now a duplicated key
```
-            'EmailMarketing' => 'Email Marketing',
 +            'EmailTemplates' => 'Email - Templates',
              'EmailTemplates' => 'Email Templates',
```

Reply:
sorry about that I'll fix it straight away

Reply:
thanks for letting me know

Reply:
And the EmailMarketing key, is to be removed?

Reply:
I've added it back, I did the changes manually and missed those. Thanks for looking out for this.